packages/patrol/example/ios/fastlane/Appfile

```diff
@@ -0,0 +1,6 @@
+app_identifier("pl.leancode.patrol.Example") # The bundle identifier of your app
+# apple_id("[[APPLE_ID]]") # Your Apple Developer Portal username
+
+
+# For more information about the Appfile, see:
+#     https://docs.fastlane.tools/advanced/#appfile
```

Comment:
This is enough, let's remove the rest of generated stuff:

```suggestion
app_identifier("pl.leancode.patrol.Example")
```

We know what these values are and we can google the docs as well :)